guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+      $ sudo netstat -nltp   #to see that sqream is listening on ports  
+ 
+4. View Monit's service status:
+
+   .. code-block:: console
+     
+      $ sudo systemctl status monit
+
+5. If all good, enable the Monit service to start on boot: **Comment - What status result(s) are we looking for?**       
+       
+   .. code-block:: console
+     
+      $ sudo systemctl enable monit
+      
+Using Monit
+====================================
+
+Using Monit is simple and intuitive, and is used for starting processes that are not running and executing other actions in error situations. In addition, Monit has its own command syntax.
+
+For example, you can stop the **sqream3** service (being monitored by Monit) in one of the following ways:
+
+* Stopping Monit and SQream separately.
+* Stopping SQream using a Monit command.
+
+Both examples above are described in more detail below.
+
+Stopping Monit and SQream Separately
+-------------------------------------
+
+You can stop the Monit service and SQream separately as follows:
+
+.. code-block:: console
+     
+   $ sudo systemctl stop monit
+   $ sudo systemctl stop sqream3
+
+You can restart Monit as follows:
+
+.. code-block:: console
+
+   $ sudo systemctl start monit
+      
+Restarting Monit automatically restarts the SQream services.      
+
+Stopping SQream Using a Monit Command
+-------------------------------------
+
+You can stop SQream using a Monit command as follows:
+
+.. code-block:: console
+     
+   $ sudo monit stop sqream3
+
+This command spots SQream only (and not Monit).
+
+You can restart SQream as follows:
+
+.. code-block:: console
+     
+   $ sudo monit start sqream3
+       
+Monit Command Line Options
+-------------------------------------
+This section describes some of the most commonly used Monit command options:
+
+.. code-block:: console
+     
+   $ start all             - Start all services
+   $ start <name>          - Only start the named service
+   $ stop all              - Stop all services
+   $ stop <name>           - Stop the named service
+   $ restart all           - Stop and start all services
+   $ restart <name>        - Only restart the named service
+   $ monitor all           - Enable monitoring of all services
+   $ monitor <name>        - Only enable monitoring of the named service
+   $ unmonitor all         - Disable monitoring of all services
+   $ unmonitor <name>      - Only disable monitoring of the named service
+   $ reload                - Reinitialize monit
+   $ status [name]         - Print full status information for service(s)
+   $ summary [name]        - Print short status information for service(s)
+   $ report [up|down|..]   - Report state of services. See manual for options
+   $ quit                  - Kill the monit daemon process
+   $ validate              - Check all services and start if not running
+   $ procmatch <pattern>   - Test process matching pattern
+
+Using Monit to Upgrade your Version of SQream
```

Comment:
I don't like this title as it's misleading. Monit is not used to upgrade. During the upgrade, in order to avoid conflicts (such as service start), it should be used to pause/stop all running services while the manual upgrade is done. Upon successful upgrade, Monit should be used to start all SQream services